symfony/framework-bundle/3.3/Makefile

```diff
@@ -6,9 +6,22 @@ cache-warmup: cache-clear
 	@test -f bin/console && bin/console cache:warmup || echo "cannot warmup the cache (needs symfony/console)"
 .PHONY: cache-warmup
 
-serve:
-	@echo -e "\033[32;49mServer listening on http://127.0.0.1:8000\033[39m"
-	@echo "Quit the server with CTRL-C."
-	@echo -e "Run \033[32mcomposer require symfony/web-server-bundle\033[39m for a better web server"
+CONSOLE=bin/console
+$(CONSOLE):
```

Comment:
maybe : `@printf "%s could not be found. You may use an alternate binary by specifying it like this : make CONSOLE=path/to/console some_target" "$(CONSOLE)"`